### ISSUE TITLE
Improve webhook testing with new dev tools

### ISSUE THREAD

**Issue by JulianBissekkou** [appwrite/appwrite#845]
## 🚀 Feature
An easy way to trigger fake webhook events for testing webhook endpoints!

## Have you spent some time to check if this issue has been raised before?
Yes, doesn't look like there is another issue like this.

### Have you read the [Code of Conduct](https://github.com/appwrite/appwrite/blob/master/CODE_OF_CONDUCT.md)?
Yes 👍🏼

## Pitch
I was working with stripe for a while now (https://stripe.com/) and they have a great SDK for client and server. 
They also have a CLI which can be used for testing webhook events.

For example:
I wanted to create a webhook event if a customer did a specific action. 
The stripe CLI allows me to create a fake event and then I am able to send that to my webhook endpoint.

`stripe listen --forward-to http://localhost:8080/myEndpoint`
`stripe trigger name_of_event`

We don't need a CLI for appwrite, but maybe some tooling to make webhook testing a little easier. 
Maybe you can share some thoughts on this  :relaxed:

**Comment by eldadfux** [comment]
I really like this feature. I think it will improve onboarding and dev experience.


I can think of two options to get this done:

1. Add a trigger endpoint which will be available as a button on the UI, with the option to trigger any system supported event with mock data (can be auto-generated from our Models objects).
2. Expose the webhooks API completely just as any other Appwrite API and allow to use the new endpoint from API, UI and upcoming [Appwrite CLI](https://github.com/appwrite/sdk-generator/pull/94).

Second option will force us to move all the webhooks data objects from the console DB to the project specific DB. The endpoint will only be accessible on server integration with an API key and new custom scopes.

**Comment by JulianBissekkou** [comment]
I think the second option would be better for future features such as the CLI. 
Another use case: I can create a unit test on my server that checks that my webhook endpoint works as expected with the appwrite server.
The test would create a webhook, executes a mocked event, and removed the webhook after the test is done.

If I upgrade appwrite to a newer version, I can just run the tests and make sure that everything works with the new appwrite version 👍 

I did a similar thing with our stripe webhooks and it was quite handy after we migrated to the next major version of the stripe sdk.
